packages/react-native/scripts/cocoapods/utils.rb

```diff
@@ -59,15 +59,23 @@ def self.turn_off_resource_bundle_react_core(installer)
     end
 
     def self.exclude_i386_architecture_while_using_hermes(installer)
+        isUsingHermes = self.has_pod(installer, 'hermes-engine')
```

Comment:
nit: if this is false, we can return immediately and avoid the two `for-each`s

Reply:
done ✅